### PR TITLE
Publish turn notification when game is started

### PR DESCRIPTION
**Pull request by daniel-sousa-me**

### CODE REVIEW

**Comment by daniel-sousa-me** [comment]
Even though this is not run in js, I changed the "any?" to "empty?" for consistency. Can remove that if preferred.

File: routes/game.rb [hunk 1]
```diff
@@ -145,7 +145,19 @@ class Api
               halt(400, 'Player count not supported')
             end
 
-            set_game_state(game, engine, users)
+            acting = set_game_state(game, engine, users)
+
+            unless acting.empty?
```
Comment (tobymao) [inline routes/game.rb:150]: why would this ever be empty?

File: routes/game.rb [hunk 2]
```diff
@@ -145,7 +145,19 @@ class Api
...
+              MessageBus.publish(
```
Comment (tobymao) [inline routes/game.rb:151]: game_url looks changed here maybe you should turn this into a function

File: routes/game.rb (hunk 1 above)
Comment (daniel-sousa-me) [reply routes/game.rb:150]: I don't know. Just copied what was above. But indeed it seems highly unlikely that nobody is acting when the games is starting.

File: routes/game.rb (hunk 2 above)
Comment (daniel-sousa-me) [reply routes/game.rb]: Indeed. I don't know how that could have happened (there was no change on this line..)

**Review by tobymao — APPROVED** [review]

**Pull request merged**